src/store/modules/user/actions.ts

```diff
@@ -58,6 +58,15 @@ const actions: ActionTree<UserState, RootState> = {
       if (resp.data.userTimeZone !== localTimeZone) {
         emitter.emit('timeZoneDifferent', { profileTimeZone: resp.data.userTimeZone, localTimeZone});
       }
+      const bopisPreferenceResp = await UserService.getShippingOrderPreference({
```

Comment:
```suggestion
      const bopisPreferenceResp = await UserService.getUserPreference({
```

I think we can name it to getUserPreference, setUserPreference and also update the name for mutation as we are getting the user preference.

Reply:
@adityasharma7 Sir thoughts?

Reply:
+1

Reply:
Should we also rename `bopisPreferenceResp` to `userPreferenceResp`?

Reply:
Updated variables, services and mutations name.